src/components/UI/SourceSelect/SourceSelect.theme.js

```diff
@@ -35,27 +50,45 @@ export const SourceSelectTheme = ({children}) => {
       MuiPaper: {
         styleOverrides: {
           root: {
-            marginTop: '4px',
-            border: `1px solid ${colorWhite}`,
-            borderRadius: '8px',
-            backgroundColor: colorDarkSlateBlue,
-            color: colorWhite
+            backgroundColor: colorSpaceCadet,
+            color: colorWhite,
+            borderRadius: '8px'
           }
         }
       },
       MuiList: {
         styleOverrides: {
           root: {
-            padding: '0'
+            width: '240px',
+            padding: '8px 16px'
+          }
+        }
+      },
+      MuiListSubheader: {
+        styleOverrides: {
+          root: {
+            padding: 0,
+            backgroundColor: 'unset',
+            lineHeight: '24px',
+            color: colorLightSteelBlue,
+            fontFamily: primaryFont,
+            letterSpacing: '0.01em'
           }
         }
       },
       MuiMenuItem: {
         styleOverrides: {
           root: {
             fontFamily: primaryFont,
-            '&.Mui-selected': {
-              backgroundColor: 'rgba(255, 255, 255, 0.1) !important'
+            fontSize: '14px',
+            padding: '4px',
+            borderRadius: '8px',
+            margin: '8px 0',
+            '&.Mui-selected, &:hover': {
+              backgroundColor: `${colorSpaceCadet2} !important`
```

Comment:
it's working without the `!important`

Reply:
@galuzan-starkware it's not working on hovering the selected w/o !important